src/website/src/app/documentation/demos/accordion/accordion.demo.html

```diff
@@ -0,0 +1,38 @@
+<clr-doc-wrapper [ng]="ng" [ui]="ui" [title]="title" [newLayout]="newLayout">
+  <article>
+    <h5 class="component-summary" id="stepper-header">
+      A accordion allows generic content to be collapsed and allow users to expand to show more detail.
+    </h5>
+
+    <div id="design-guidelines">
+      <h3>Design Guidelines</h3>
+
+      <h3>Usage</h3>
+      <p>
+        Accordions are designed for use in the main content area. Use a accordion when you want to progressively
```

Comment:
Nit: I believe it should be "an accordion". Not?

Reply:
fixed